app/assets/stylesheets/partials/status-panel.css.scss

```diff
@@ -168,17 +168,17 @@
     padding: 0;
     margin-bottom: 0;
     overflow: hidden;
+    text-overflow: ellipsis;
     max-height: 500px;
+    max-width: 99.9%; // HACK; Prevent ellipsis overflow of characters
```

Comment:
I am honestly _more_ confused now with this comment than I was before.  @JoshFabian it's up to you if this hack is up to your CSS standards.